functions.py

```diff
@@ -67,3 +67,76 @@ def signUpUser():
         print("Sorry, that username is already taken")
         signUpUser()
 
+def handleLogin():
+    print("Do you want to log in or start a new account?\n(Enter the number of your choice)\n")
+    print("(1) Log In")
+    print("(2) Start New Account\n")
+    option = getUserInput("> ")
+    if option == "1":
+        loginUser()
+
+    elif option == "2":
+        signUpUser()
+
+    else:
+        print("Please enter a valid option\n")
+        handleLogin()
+
+def welcomeMessage():
+    print("\n\n")
+    print("##################################################")
+    print("# Welcome to Passman!")
+    print("##################################################\n\n")
+
+def listServicesPrompt():
```

Comment:
@nosnixster Starting at this line are some functions for you to fill in. See the showMenu() function to get a feel for how they are being called.